addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -603,7 +603,8 @@ body.editor_enable.editor_has_snippets {
 
         // Checkboxes
         we-button.o_we_checkbox_wrapper.o_we_user_value_widget {
-            padding: 0;
+            width: fit-content;
```

Comment:
This is not supported by all browsers, I made a different fix here: https://github.com/odoo/odoo/pull/59240